web/src/components/apps/AppDetailPage.tsx

```diff
@@ -85,69 +90,120 @@ class AppDetailPage extends Component<Props, State> {
       redeployVersionErrMsg: "",
       requiredKotsUpdateMessage: "",
       selectedWatchName: "",
-    };
-  }
+    }
+  );
+
+  const history = useHistory();
+  const params = useParams<KotsParams>();
+  const [currentApp, setCurrentApp] = useState(
```

Comment:
i have a hook for the current app but i'm not using it here yet. I need to wire up the `useApps` stuff so the `currentApp` will be accurate. will do that in the next pr